intern/remote/boss/remote.py

```diff
@@ -71,6 +71,14 @@ def __init__(self, cfg_file_or_dict=None, version=None):
         self._init_metadata_service(version)
         self._init_volume_service(version)
 
+    def __repr__(self):
+        """
+        Stringify the Remote.
+
+        Returns a representation of the BossRemote that lists the host.
+        """
+        return f"<intern.remote.BossRemote [{self._config['Default']['host']}]>"
```

Comment:
I did not know about the f operator for string formatting. Can’t wait to start using it!

However, I don’t think 2.7 supports and I don’t think we’ve mandated that intern users use 3.6+ when using 3.x.

Reply:
Great point, just fixed this. Completely forgot this was a recent addition to the language!